docs/source/docs/quick-start/camera-calibration.md

```diff
@@ -8,7 +8,7 @@ If you’re not using cameras in 3D mode, calibration is optional, but it can st
 
 ## Print the Calibration Target
 
-- Downloaded from our [demo site](https://demo.photonvision.org/#/cameras), or directly from your coprocessors cameras tab.
+- Downloaded from our [demo site](http://photonvision.global/), or directly from your coprocessors cameras tab.
```

Comment:
```suggestion
- Downloaded from our [demo site](http://photonvision.global/#/cameras), or directly from your coprocessors cameras tab.
```

Reply:
it goes to the camera tab like the old url did

Reply:
I dont think we had a good reason for doing that previously did we?

Reply:
not really. it's because the generate board button is on that page

Reply:
It goes directly to the generate board button, which is where you download it from.